tests/topotato/test_bgp_default-originate_route-map_match_set.py

```diff
@@ -0,0 +1,123 @@
+from topotato import *
+"""
+Test if default-originate works with match operations.
+And verify if set operations work as well.
+
+"""
+
+@topology_fixture()
+def allproto_topo(topo):
+  """
+  [ r1 ]
+    |
+  { s1 }
+    |
+  [ r2 ]
+  
+  """
+  topo.router("r1").lo_ip4.append("172.16.255.254/32")
+  topo.router("r1").iface_to("s1").ip4.append("192.168.255.1/24")
+  topo.router("r2").iface_to("s1").ip4.append("192.168.255.2/24")
+
+class Configs(FRRConfigs):
+  routers = ["r1", "r2"]
+    
+  zebra = """
+    #% extends "boilerplate.conf"
+    #% block main
+    #%   if router.name == 'r1'
+    interface lo
+     ip address {{ routers.r1.lo_ip4[0] }} 
+    !
+    ip route 192.168.13.0./24 Null0

```

Comment:
Typo here (extra dot on the IP)

… also TODO for me: this creates a warning in the FRR log, but topotato doesn't check for it :disappointed: … need the negative assertions for this …